support/reattach-pv/README.md

```diff
@@ -39,6 +51,34 @@ go build
 ./reattach-pv --elasticsearch-manifest elasticsearch.yml
 ```
```

Comment:
Instead of repeating the whole tool usage, we could just improve the example. I think we can skip the `go build`.

------------------------------

Example assuming `cluster-A` was accidently deleted:

```
# re-create the cluster with the same name
./reattach-pv --elasticsearch-manifest cluster-A.yml --dry-run

# re-create the cluster with a new name
reattach-pv --elasticsearch-manifest cluster-B.yml --old-elasticsearch-name cluster-A --dry-run

# if everything seems ok, then, execute again without the dry-run flag
```